### PR TITLE
Full editing pass

### PR DESCRIPTION
**Pull request by Buzzardo**
I corrected spelling and grammar and gave it a voice (flat, corporate) consistent with our other docs. I added a couple TODOs where I thought another sample would help.

### CODE REVIEW

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -181,20 +187,21 @@ public class SampleTask {
 
 		@Override
 		public void run(String... strings) throws Exception {
-			System.out.println("Hello World!");
+			System.out.println("Hello, World!");
```
Comment (cppwfs) [inline spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: So strict on a `Hello World` app.  We're taking over the world ;-)

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -181,20 +187,21 @@ public class SampleTask {
...
+Boot specifics, see the
+http://docs.spring.io/spring-boot/docs/current/reference/html/[Spring Boot reference documentation].
 
-We'll also need to create an `application.properties` in `src/main/resources`.  We'll
-configure two properties in it: the application name (which is translated to the task name)
-and we'll set the logging for spring cloud task to `DEBUG` so that we can see what's going
-on:
+We also need to create an `application.properties` file in `src/main/resources`. We need
+to configure two properties in `application.properties`: We need to set the application
+name (which is translated to the task name), and we need to set the logging for spring
```
Comment (cppwfs) [inline spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: Spring Cloud Task (caps) just for consistency.

File: spring-cloud-task-docs/src/main/asciidoc/batch.adoc [hunk 3]
```diff
@@ -4,37 +4,40 @@
...
+`spring-cloud-task-batch` jar on the classpath. The listener is injected into all jobs
+that meet those conditions.
 
 [[batch-association-override]]
 === Overriding the TaskBatchExecutionListener
 
-To prevent the listener from being injected into any batch jobs within the current context,
-the autoconfiguration can be disabled via standard Spring Boot mechanisms.
+To prevent the listener from being injected into any batch jobs within the current
+context, you can disagle the autoconfiguration by using standard Spring Boot mechanisms.
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/batch.adoc]: Yup.

File: spring-cloud-task-docs/src/main/asciidoc/batch.adoc [hunk 4]
```diff
@@ -43,29 +46,31 @@ public TaskBatchExecutionListenerBeanPostProcessor batchTaskExecutionListenerBea
 
 	return postProcessor;
 }
-```
-NOTE: A sample batch application can be found in the samples module
-of the Spring Cloud Task Project
+----
+
+NOTE: You can find a sample batch application in the samples module of the  Spring Cloud
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/batch.adoc]: Yup.

File: spring-cloud-task-docs/src/main/asciidoc/batch.adoc [hunk 5]
```diff
@@ -88,53 +93,62 @@ public PartitionHandler partitionHandler(TaskLauncher taskLauncher,
...
 		new DeployerStepExecutionHandler(this.context, jobExplorer, this.jobRepository);
 
 	return handler;
 }
-```
-NOTE: A sample remote partition application can be found in the samples module
-of the Spring Cloud Task Project
+----
+
+NOTE: You can find a sample remote partition application can be found in the samples module
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/batch.adoc]: Yup

File: spring-cloud-task-docs/src/main/asciidoc/batch.adoc [hunk 6]
```diff
@@ -146,45 +160,53 @@ following dependency for the Spring Cloud Yarn Deployer
...
+
+* When configuring the partition handler, do not add any command line arguments to the
+`CommandLineArgsProvider`. You need to avoid adding arguments, because Chronos adds the
+command line args to the Mesos ID. If the command line arguments contains characters such
+as `/` or `:` when launching the partition on Mesos, the partition fails to start .
+
+=== Notes on Developing a Batch-partitioned Application for the Cloud Foundry Platform
+
+* When deploying partitioned apps on the Cloud Foundry platform, you must use the
+following dependencies for the Spring Cloud Cloud Foundry Deployer:
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/batch.adoc]: But it's important! ;)

Yup....

File: spring-cloud-task-docs/src/main/asciidoc/batch.adoc [hunk 7]
```diff
@@ -226,37 +248,38 @@ spring_cloud_deployer_cloudfoundry_username=admin
...
-the `spring.cloud.task.batch.commandLineRunnerOrder` property.
-To have your task return the exit code based on the result of the batch job
-execution, you will need to write your own `CommandLineRunner`.
-
+of `FAILED`, set `spring.cloud.task.batch.failOnJobFailure` to `true`. Then the exit code
+can be 1 (the default) or be based on the
+https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-spring-application.html#boot-features-application-exit[specified
+`ExitCodeGenerator`])
+
+This functionality usee a new `CommandLineRunner` that replaces the one provided by Spring
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/batch.adoc]: Yup

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc [hunk 8]
```diff
@@ -4,54 +4,57 @@
...
 [[getting-started-developing-first-task]]
-== Developing your first Spring Cloud Task application
+== Developing Your First Spring Cloud Task Application
 
-A good place to start is with a simple "Hello World!" application so we'll create the
-Spring Cloud Task equivalent to highlight the features of the framework.  We'll use Apache
-Maven as a build tool for this project since most IDEs have good support for it.
+A good place to start is with a simple "`Hello, World!`" application, so we create the
+Spring Cloud Task equivalent to highlight the features of the framework. Most IDEs have
+good support for Apache Maven, so we use it as the build tool for this project since.
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: Yup.

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc [hunk 9]
```diff
@@ -4,54 +4,57 @@
...
-If you’re looking to solve a specific problem; check there first.  You can shortcut the
-steps below by going to start.spring.io and creating a new project. This will
-automatically generate a new project structure so that you can start coding right the way.
+NOTE: The spring.io web site contains many https://spring.io/guides[“`Getting Started`”
+guides] that use Spring Boot. If you need to solve a specific problem, check there first.
+You can shortcut the following steps by going to the
+http://start.spring.io/[Spring Initializr] and creating a new project. Doing so
+automatically generates a new project structure so that you can start coding right away.
 Check the documentation for more details.
+// TODO: What documentation? The Spring Initalizr doesn't have any documentation links.
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: OK. I'll reword that.

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc [hunk 10]
```diff
@@ -65,14 +68,15 @@ Apache Maven 3.2.3 (33f8c3e1027c3ddde99d3cdebad2656a31e8fdf4; 2014-08-11T15:58:1
 Maven home: /usr/local/Cellar/maven/3.2.3/libexec
 Java version: 1.8.0_31, vendor: Oracle Corporation
 
-NOTE: This sample needs to be created in its own folder.  Subsequent instructions assume
-you have created a suitable folder and that it is your "current directory".
+NOTE: This sample needs to be created in its own folder. Subsequent instructions assume
+you have created a suitable folder and that it is your "`current directory.`"
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: That combination creates curly quotes. I'll verify that that's happening in admonitions and, if not, bash on the Asciidoc syntax till it does.

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: I'm firmly in the "details matter" camp - taking over the world one comma at a time! ;)

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc (hunk 2 above)
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: Yup

File: spring-cloud-task-docs/src/main/asciidoc/features.adoc [hunk 11]
```diff
@@ -226,63 +218,66 @@ The `TaskConfigurer` allows the configuration of three main components:
...
 [[features-task-name]]
 === Task Name
 
-In most cases, the name of the task will be the application name as configured via Spring
-Boot.  However, there are some cases, where you may want to map the run of a task to a
-different name.  Spring Data Flow is an example of this (where you want the task to be run
-with the name of the task definition).  Because of this, we offer the ability to customize
-how the task is named via the `TaskNameResolver` interface.
+In most cases, the name of the task is the application name as configured in Spring
+Boot. However, there are some cases where you may want to map the run of a task to a
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/features.adoc]: Got it

File: spring-cloud-task-docs/src/main/asciidoc/features.adoc [hunk 12]
```diff
@@ -297,69 +292,70 @@ marking the final state of the task.
...
-a task with the same <<features-task-name, task name>> and
-`spring.cloud.task.singleInstanceEnabled=true` the task will fail with the following
-error message `Task with name "application" is already running.`  The default
-for `spring.cloud.task.singleInstanceEnabled` is `false`.
-```
- spring.cloud.task.singleInstanceEnabled=<true or false>
-```
-In order for this feature to be used you must include the following Spring
-Integration dependencies to your application:
+Spring Cloud Task Instances let you establish that only one task with a given task name
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/features.adoc]: I think it works either way, but fewer words is better.

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc [hunk 13]
```diff
@@ -4,54 +4,57 @@
...
-capabilities that allow short lived JVM processes to be executed on demand in a production
+Spring Cloud Task makes it easy to create short-lived microservices. It provides
+capabilities that let short lived JVM processes be executed on demand in a production
 environment.
 
 [[getting-started-system-requirements]]
 == System Requirements
 
-You need Java installed (Java 7 or better, we recommend Java 8) and to build you need to have Maven installed as well.
+You need to have Java installed (Java 7 or better -- we recommend Java 8). To build, you
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: Got it

File: spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc (hunk 10 above)
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc]: Update: It is working correctly. "`something`" puts curly quotation marks around "something".

**Comment by cppwfs** [comment]
LGTM

**Comment by cppwfs** [comment]
Squashed Merged.

**Pull request closed without merging**